packages/sdks/snippets/gen1-remix/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@snippet/gen1-remix",
+  "private": true,
+  "sideEffects": false,
+  "type": "module",
+  "scripts": {
+    "build": "remix vite:build",
+    "dev": "remix vite:dev",
+    "lint": "eslint --ignore-path .gitignore --cache --cache-location ./node_modules/.cache/eslint .",
+    "start": "remix-serve ./build/server/index.js",
+    "typecheck": "tsc",
+    "test": "SERVER_NAME=gen1-remix yarn nx test:snippet @sdk/tests",
+    "serve": "remix-serve ./build/server/index.js"
+  },
+  "dependencies": {
+    "@builder.io/react": "workspace:*",
+    "@emotion/core": "10.0.17",
```

Comment:
had to install `@emotion/core` as a dependency here as it was not able to correctly bundle the `@emotion/core` from `@builder.io/react` linked package [was trying to use the CJS build], but we would not need this when we are installing `@builder.io/react` as a package

Remix app deployed: https://github.com/sidmohanty11/builder-remix-gen1-example

Reply:
can you add this as a note in the README. it's an important gotcha for us maintaining it.

Reply:
added